fdbserver/BlobWorker.actor.cpp

```diff
@@ -952,6 +952,84 @@ ACTOR Future<BlobFileIndex> writeDeltaFile(Reference<BlobWorkerData> bwData,
 	}
 }
 
+ACTOR Future<BlobFileIndex> writeEmptyDeltaFile(Reference<BlobWorkerData> bwData,
```

Comment:
nit - maybe a single line comment to explain why we want to generate empty delta files?

Reply:
Yeah specifically the #9243 pr's force flushing seems to cause this logic to happen pretty often.
And I agree with adding comments, but I'm a bit afraid to rerun CI as it's been so flaky haha.